packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx

```diff
@@ -148,47 +150,83 @@ export function BalanceWithCarryover({
                 <span style={{ fontWeight: 'bold' }}>
                   {getDifferenceToGoal(balanceValue) === 0 ? (
                     <span style={{ color: theme.noticeText }}>
-                      {t('Fully funded')}
+                      <Trans>Fully funded</Trans>
                     </span>
                   ) : getDifferenceToGoal(balanceValue) > 0 ? (
                     <span style={{ color: theme.noticeText }}>
-                      {t('Overfunded ({{amount}})', {
-                        amount: format(
-                          getDifferenceToGoal(balanceValue),
-                          'financial',
-                        ),
-                      })}
+                      <Trans>
+                        Overfunded (
+                        {{
+                          amount: format(
+                            getDifferenceToGoal(balanceValue),
+                            'financial',
+                          ),
+                        }}
+                        )
+                      </Trans>
                     </span>
                   ) : (
                     <span style={{ color: theme.errorText }}>
-                      {t('Underfunded ({{amount}})', {
-                        amount: format(
-                          getDifferenceToGoal(balanceValue),
-                          'financial',
-                        ),
-                      })}
+                      <Trans>
+                        Underfunded (
+                        {{
+                          amount: format(
+                            getDifferenceToGoal(balanceValue),
+                            'financial',
+                          ),
+                        }}
+                        )
+                      </Trans>
                     </span>
                   )}
                 </span>
                 <GoalTooltipRow>
-                  <div>{t('Goal Type:')}</div>
-                  <div>{longGoalValue === 1 ? t('Long') : t('Template')}</div>
+                  <Trans>
+                    <div>Goal Type:</div>
+                    <div>
+                      {
+                        {
+                          type: longGoalValue === 1 ? t('Long') : t('Template'),
```

Comment:
Could be worth adding [context](https://www.i18next.com/translation-function/context) for translators that "Long" is a noun we're defining rather than just a random word

Reply:
That's fair, though there is a spot in the UI for it. Not sure if it works right with our pipeline. FF to skip!

Reply:
1c652fb5a93a7cd593fc5ebbba1d7b250a4df3c0 let's give it a go!